evaluation/segmentationevaluation.py

```diff
@@ -0,0 +1,142 @@
+import argparse
+import csv
+import datetime
+import glob
+import os
+import random
+
+import h5py
+import napari
+
+from evaluation import RESULTS
+from evaluation.makestatistics import compute_stats
+
+KEYS_TO_USE = {'marker': {'data_type': 'image', 'visible': False},
+               'nuclei': {'data_type': 'image', 'visible': True},
+               'serum': {'data_type': 'image', 'visible': False},
+               'cell_segmentation': {'data_type': 'label', 'visible': True},
+               'mask': {'data_type': 'label', 'visible': False}
+               }
+
+
+def write_csv(csv_path, dataset, name, value):
+    with open(csv_path, 'a+') as f:
+        writer = csv.writer(f)
+        writer.writerow([dataset, name, value])
+
+
+def load_stack(h5_path):
+    stack = {}
+    with h5py.File(h5_path, "r") as f:
```

Comment:
If you use `util.io.open_file` instead, it will also work for n5 files (been discussing with Tischi to give that a try, because it could make his viewer smoother).

Reply:
Sure, I'll change it now